diff --git a/node_modules/paredit.js/lib/reader.js b/node_modules/paredit.js/lib/reader.js
index 55b47ed..f4fcdb4 100644
--- a/node_modules/paredit.js/lib/reader.js
+++ b/node_modules/paredit.js/lib/reader.js
@@ -23,11 +23,11 @@
   // read logic
 
   var eosexp = {}, eoinput = {}, // flags
-      close = {'[': ']', '(': ')', '{': '}'},
+      close = {'(': ')'},
       opening = Object.keys(close),
       closing = opening.map(function(k) { return close[k]; }),
-      symRe = /[^\s\[\]\(\)\{\},"\\`@^#~]/,
-      readerSpecials = /[`@^#~]/;
+      symRe = /[^\s\(\)\\"#;,`']/,
+      readerSpecials = /['`#,@]/;
 
   function readSexp(contextStart, input, context, pos, xform) {
     var ch = input[0];
@@ -47,7 +47,7 @@
     }
 
     // 3. whitespace
-    if (/\s|,/.test(ch)) return {
+    if (/\s/.test(ch)) return {
       input: input.slice(1),
       context: context,
       pos: forward(pos, ch)
